lighthouse-core/report/html/report-styles.css

```
@@ -341,7 +341,8 @@
 .lh-audit-group a,
 .lh-category-header__description a,
 .lh-audit__description a,
-.lh-footer a {
+.lh-footer a,
+.lh-table-column-link a {
```

```suggestion
.lh-table-column--link a {
```

this should do it :)